sites/docs/src/content/docs/usage/configuration/introduction.md

```diff
@@ -0,0 +1,178 @@
+
+If you clone and manually edit an nf-core pipeline then you cannot update to more recent versions of the pipeline without overwriting your changes. You also risk moving away from the canonical pipeline and losing reproducibility.
+
+:::tip
+Configuration locations can be used to your benefit. For example, use `-profile` for your cluster configuration, `~/.nextflow/config` for your personal configuration, and `nextflow.config` in your working directory for run-specific configuration.
+:::
+
+### Parameters
+
+Parameters are pipeline specific settings that can be used to customize the execution of a pipeline.
+
+At the highest level, parameters can be customized using the command line. Any parameter can be configured on the command line by prefixing the parameter name with a double dash (--):
+
+```bash
+--<parameter>
+```
+
+Depending on the parameter type, you may be required to add additional information after your parameter flag.
+For example, you would add string parameter after the parameter flag for the `nf-core/rnaseq` `--input` and `--output` parameters.
+
+```bash
+nextflow nf-core/rnaseq --input <path/to/input> --outdir <results>
+```
+
+Every nf-core pipeline has a full list of parameters on the nf-core website. You will be shown a description and the type of the parameter when viewing these parameters. Some parameters will also have additional text to help you understand how a parameter should be used. See the [parameters page of the nf-core rnaseq pipeline](https://nf-co.re/rnaseq/3.14.0/parameters/).
+
+### Default configuration files
+
+All parameters have a default configuration that is defined using the `nextflow.config` file in the root of the pipeline directory. Many parameters are set to `null` or `false` by default and are only activated by a profile or config file.
+
+There are also several `includeConfig` statements in the `nextflow.config` file that are used to include additional config files from the `conf/` folder. Each additional `.config` file contains categorized configuration information for your pipeline execution, some of which can be optionally included as profiles:
+
+- `base.config`
+  - Included by the pipeline by default
+  - Generous resource allocations using labels
+  - Does not specify any method for software dependencies and expects software to be available (or specified elsewhere)
+- `igenomes.config`
+  - Included by the pipeline by default
+  - Default configuration to access reference files stored on AWS iGenomes
+- `modules.config`
+  - Included by the pipeline by default
+  - Module-specific configuration options (both mandatory and optional)
+- `test.config`
+  - Only included if specified as a profile
+  - A configuration profile to test the pipeline with a small test dataset
+- `test_full.config`
+  - Only included if specified as a profile
+  - A configuration profile to test the pipeline with a full-size test dataset
```

Comment:
I guess we are getting a bit more advanced than the quick_Start bit, so I would recommend again saying that 'you should not modify these'

Reply:
Added below